pkg/spidev/spidev_linux.go

```diff
@@ -301,3 +331,30 @@ func (s *SPI) SetSpeedHz(hz uint32) error {
 	}
 	return nil
 }
+
+// ID gets the 24-bit id as an int
+func (s *SPI) ID() (int, error) {
```

Comment:
The ID function should probably go into the Flash struct since it is specific to flash chips.

Reply:
I'm mixed on this. To get the ID, you have to use SPI. No way around it. So it's the ID you get by using low level SPI -- if you look, it's a separate opcode. I think this has to stay here. 
We could rename ID to FlashID, but I really don't think that's necessary.
Another thing I'm not sure about -- don't all SPI devices have some sort of ID? 

Note they did the one byte manufacturer ID, which I thought we'd all learned was a bad idea.

Reply:
Besides, it's going ot be kind of in the Flash struct, via Chip.
Should we just embed Chip?

Reply:
I embedded chip